src/systems/spawner.rs

```diff
@@ -77,18 +77,26 @@ impl<'s> System<'s> for CreatureSpawnerSystem {
     }
 }
 
+//
+//
 // For debugging purposes this system sends spawn events regularly
 #[derive(Default)]
 pub struct DebugSpawnTriggerSystem {
     timer_to_next_spawn: f32,
+    bee_timer: f32,
 }
 
 impl<'s> System<'s> for DebugSpawnTriggerSystem {
-    type SystemData = (Write<'s, EventChannel<CreatureSpawnEvent>>, Read<'s, Time>);
+    type SystemData = (
+        Write<'s, EventChannel<CreatureSpawnEvent>>,
+        Read<'s, Time>,
+        Read<'s, CreaturePrefabs>,
+    );
 
-    fn run(&mut self, (mut spawn_events, time): Self::SystemData) {
+    fn run(&mut self, (mut spawn_events, time, _creature_prefabs): Self::SystemData) {
         let delta_seconds = time.delta_seconds();
         self.timer_to_next_spawn -= delta_seconds;
+        self.bee_timer -= delta_seconds;
```

Comment:
As mentioned above, it would be great if you could move this into a separate PR. In addition, you could write a separate system so that we can enable/disable bee spawning by registering/unregistering systems.

Reply:
I removed the bee from this PR.